lib/Net/Twitter/Core.pm

```diff
@@ -266,10 +272,10 @@ sub _parse_result {
         die Net::Twitter::Error->new(twitter_error => $obj, http_response => $res);
     }
 
-    return $obj if $res->is_success && defined $obj;
+    return $obj if $res->is_success;
```

Comment:
This concerns me. We probably need to handle it differently.

One of the most frustrating Twitter errors is a 200 OK with HTML content. It happens to frequently to be ignored. Usually, you get HTML for the twitter.com homepage. So, it's an ops problem on Twitter's side.

Perhaps we should change line 259:
```perl
my $obj = length $content ? try { $self->_from_json($content) } : {};
```
That will give us a defined $obj and still allow us to handle the weird 200 OK with HTML responses.